bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcUniqueConfig.scala

```diff
@@ -37,9 +37,13 @@ case class ItcUniqueConfig(count: Int, labels: String, configs: NonEmptyList[Con
   /** TReCS and Michelle have an optional total time on source. */
   private val totalTime = Option(config.getItemValue(INST_TIME_ON_SRC_KEY)).map(_.asInstanceOf[Double])
 
-  /** Instruments other than TReCS and Michelle have a defined exposure time per image. */
-  private val singleTime = Option(config.getItemValue(INST_EXP_TIME_KEY)).map(_.asInstanceOf[Double])
-
+  /** Instruments other than TReCS and Michelle have a defined exposure time per
+    * image.  In the case of GMOS, this may include nod and shuffle steps so
+    * full exposure time is preferred.
+    */
+  private val singleTime: Option[Double] =
+    if (ItcUniqueConfig.isGmos(config)) Some(GmosExposureTime.asDoubleSecs(config))
+    else Option(config.getItemValue(INST_EXP_TIME_KEY)).map(_.asInstanceOf[Double])
```

Comment:
Above you're handling the case where it's something other than a `Double` but you're assuming it here.

Reply:
The `else` clause is just "whatever it was doing before".  `totalTime` and `singleTime` were just cast before and I was trying to disturb the ITC stuff as little as possible.

Reply:
Alright.